Build error on Fedora 35

I am getting the following error while trying to build verible on fedora 35:

```
➜  verible git:(master) bazel build -c opt //...
Starting local Bazel server and connecting to it...
INFO: Analyzed 1864 targets (111 packages loaded, 5362 targets configured).
INFO: Found 1864 targets...
bazel: Entering directory `/home/jordi/.cache/bazel/_bazel_jordi/13967d7c87a057fae689ac53148986f9/execroot/com_google_verible/'
ERROR: /home/jordi/.cache/bazel/_bazel_jordi/13967d7c87a057fae689ac53148986f9/external/m4_v1.4.18/gnulib/BUILD.bazel:192:11: Compiling gnulib/lib/c-stack.c [for host] failed: (Exit 1): gcc failed: error executing command /usr/bin/gcc -U_FORTIFY_SOURCE -fstack-protector -Wall -Wunused-but-set-parameter -Wno-free-nonheap-object -fno-omit-frame-pointer -g0 -O2 '-D_FORTIFY_SOURCE=1' -DNDEBUG -ffunction-sections ... (remaining 25 argument(s) skipped)

Use --sandbox_debug to see verbose messages from the sandbox
In file included from /usr/include/signal.h:328,
                 from external/m4_v1.4.18/gnulib/lib/c-stack.c:49:
external/m4_v1.4.18/gnulib/lib/c-stack.c:55:26: error: missing binary operator before token "("
   55 | #elif HAVE_LIBSIGSEGV && SIGSTKSZ < 16384
      |                          ^~~~~~~~
external/m4_v1.4.18/gnulib/lib/c-stack.c:139:8: error: variably modified 'buffer' at file scope
  139 |   char buffer[SIGSTKSZ];
      |        ^~~~~~
bazel: Leaving directory `/home/jordi/.cache/bazel/_bazel_jordi/13967d7c87a057fae689ac53148986f9/execroot/com_google_verible/'
INFO: Elapsed time: 10.128s, Critical Path: 3.96s
INFO: 42 processes: 13 internal, 29 linux-sandbox.
FAILED: Build did NOT complete successfully
```
My toolset version:
```
➜  ~ g++ --version
g++ (GCC) 11.2.1 20210728 (Red Hat 11.2.1-1)
Copyright (C) 2021 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

➜  ~ bazel --version
bazel 4.2.1
```

Any idea why this happens?
I was able to build in RHEL8 but not in Fedora35, which is strange.

--- a/.github/workflows/verible-ci.yml
+++ b/.github/workflows/verible-ci.yml
@@ -132,7 +132,7 @@ jobs:
       run: |
         set -x
         source ./.github/settings.sh
-        sudo apt -qq -y install clang-10
+        sudo apt -qq -y install clang-10 flex bison
         ./.github/bin/set-compiler.sh 9
         ./.github/bin/install-bazel.sh
         ./.github/bin/install-python.sh

--- a/README.md
+++ b/README.md
@@ -45,6 +45,8 @@ Verible's code base is written in C++.
 To build, you need the [bazel] build system and a C++17 compatible compiler
 (e.g. >= g++-9), as well as python3.
 
+Linux systems also require the following tools: m4, flex, bison.
+
 Use your package manager to install the dependencies; on a system with
 the nix package manager simply run `nix-shell` to get a build environment.
 

--- a/bazel/bison.bzl
+++ b/bazel/bison.bzl
@@ -32,10 +32,12 @@ def genyacc(
         outs = [header_out, source_out] + extra_outs,
         cmd = select({
             "@platforms//os:windows": "$(BISON) --defines=$(location " + header_out + ") --output-file=$(location " + source_out + ") " + " ".join(extra_options) + " $<",
+            "@platforms//os:linux": "bison --defines=$(location " + header_out + ") --output-file=$(location " + source_out + ") " + " ".join(extra_options) + " $<",
             "//conditions:default": "M4=$(M4) $(BISON) --defines=$(location " + header_out + ") --output-file=$(location " + source_out + ") " + " ".join(extra_options) + " $<",
         }),
         toolchains = select({
             "@platforms//os:windows": ["@rules_bison//bison:current_bison_toolchain"],
+            "@platforms//os:linux": [], # use host m4/bison
             "//conditions:default": [
                 "@rules_bison//bison:current_bison_toolchain",
                 "@rules_m4//m4:current_m4_toolchain",

--- a/bazel/flex.bzl
+++ b/bazel/flex.bzl
@@ -26,10 +26,12 @@ def genlex(name, src, out):
         outs = [out],
         cmd = select({
             "@platforms//os:windows": "$(FLEX) --outfile=$@ $<",
+            "@platforms//os:linux": "flex --outfile=$@ $<",
             "//conditions:default": "M4=$(M4) $(FLEX) --outfile=$@ $<",
         }),
         toolchains = select({
             "@platforms//os:windows": ["@rules_flex//flex:current_flex_toolchain"],
+            "@platforms//os:linux": [], # use host m4/flex tools
             "//conditions:default": [
                 "@rules_flex//flex:current_flex_toolchain",
                 "@rules_m4//m4:current_m4_toolchain",

--- a/verilog/parser/BUILD
+++ b/verilog/parser/BUILD
@@ -128,6 +128,7 @@ cc_library(
         "//conditions:default": [
             "-Wno-implicit-fallthrough",
             "-Wno-type-limits",
+            "-Wno-unreachable-code",
         ],
     }),
     deps = [

--- a/verilog/parser/BUILD
+++ b/verilog/parser/BUILD
@@ -122,7 +122,7 @@ record_recovered_syntax_errors(
 cc_library(
     name = "verilog_y_cc",
     srcs = ["verilog-final.tab.cc"],
-    hdrs = ["verilog_parse_interface.h"],
+    hdrs = ["verilog_parse_interface.h", "verilog.tab.hh"],
     copts = select({
         "@platforms//os:windows": [],
         "//conditions:default": [

